<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FireNET Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="connectBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="port">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
    <property name="value">
     <number>3322</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="loginTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>admin@noname.com</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>qwerty</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nicknameTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Administrator</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="modelTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>objects/characters/admin.cdf</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createProfileBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Create profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getProfileBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get profle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getShopBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get shop items</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buyItemBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Buy item</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="itemNameTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pistol</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="friendNameTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>AfroStalin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFriendBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add friend</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeFriendBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove friend</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getServerBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get game server</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mapNameTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>map</string>
    </property>
   </widget>
   <widget class="QListWidget" name="logWindow">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>461</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>450</y>
      <width>461</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear log window</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove item</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverIP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run stress test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopStressTest">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>460</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop stress test</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="chatTxt">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendGlobalChatBtn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send global</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendPrivateBtn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send private</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Chat system</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Log window</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>20</width>
      <height>381</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>100</y>
      <width>20</width>
      <height>381</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>470</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="gamerulesTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>gamerules</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverNameTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>name</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="9"/>
 <resources/>
 <connections/>
</ui>
